Source/Scene/JsonMetadataTable.js

```diff
@@ -15,6 +15,9 @@ import MetadataEntity from "./MetadataEntity.js";
  * @constructor
  * @private
  */
+
+const emptyClass = {};
```

Comment:
Maybe put a comment here along the lines of "JsonMetadataTable is an older type of metadata table that does not have a class definition"